### PR TITLE
report: prevent breaking words in opportunity text

### PR DESCRIPTION
**Pull request by adamraine**
Fixes https://github.com/GoogleChrome/lighthouse/issues/14326

### CODE REVIEW

**Comment by paulirish** [comment]
weirdest thing. i also made a branch for this today.

slightly diff impl:
image

will review later…

**Comment by brendankenny** [comment]
Separate from the formatting: do we prefer `s` over `sec`? `sec` seems kind of weird to me as an abbreviation, honestly

**Comment by brendankenny** [comment]
> Separate from the formatting: do we prefer `s` over `sec`? `sec` seems kind of weird to me as an abbreviation, honestly

we can switch from `unitDisplay: 'short'` to `unitDisplay: 'narrow'` to get `'s'` [here](https://github.com/GoogleChrome/lighthouse/blob/a08598fc1273f87ad52f9bb17ecf0e1733287374/report/renderer/i18n.js#L180-L183), but then 150ms becomes `0.15s` (no space, breaking or otherwise, before `'s'`). Kind of annoying but fine?

File: report/assets/styles.css [hunk 1]
```diff
@@ -873,7 +873,6 @@
 
 .lh-audit--load-opportunity .lh-audit__display-text {
   text-align: right;
-  flex: 0 0 calc(3 * var(--report-font-size));
```
Comment (alexnj) [inline report/assets/styles.css:876]: This one potentially has the side effect of the right end of spark lines being misaligned between opportunities (without a fixed and reserved space for each)?

Comment (alexnj) [reply report/assets/styles.css:876]: image

Comment (adamraine) [reply report/assets/styles.css:876]: Yeah good point, that's probably why this line was here in the first place. Bumping to `4 *` is probably best and then paul's suggestion just so a potential break doesn't happen in the middle of the word.

**Review by alexnj — APPROVED** [review]

**Pull request merged**